Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -50,6 +50,14 @@ def change_rpath(file, old_prefix, new_prefix)
     file.patch!(interpreter: updated[:interpreter], rpath: updated[:rpath])
   end
 
+  def relocate_build_prefix(old_cellar, new_prefix, new_cellar)
+    elf_files.each do |file|
+      next unless file.elf?
```

Comment:
Isn't `#elf?` always true for `elf_files`?

Reply:
I agree that these checks are redundant so I'll remove them.